phpstan.neon.dist

```diff
@@ -1,5 +1,5 @@
 includes:
-	- build-cs/vendor/phpstan/phpstan/conf/bleedingEdge.neon
+	- phar://phpstan.phar/conf/bleedingEdge.neon
```

Comment:
No idea why PHPStan didn't complain before, but we need to update the config file here to ignore any errors about `str_contains` and similar, as WP polyfills those.

Example:

https://github.com/WordPress/plugin-check/blob/6721eb4d281bf39469289590233277012e66807a/phpstan.neon.dist#L17-L27

Reply:
PHPStan takes platform PHP into account if it's present when it initialize the PHP parser - https://github.com/phpstan/phpstan-src/blob/aa801e413e81c8552878a5d7ef418f00532c6f5b/src/Php/PhpVersionFactoryFactory.php#L36

Reply:
Ah that makes sense, it was missing before.